content/guides/kubernetes-mtls/index.md

```diff
@@ -221,26 +227,28 @@ spec:
 kubectl apply -f mtls-demo.yaml
 ```
 
-The above Deployment definition of a Spring Boot client application relies on Redis Service resource being available on
+The above Deployment definition of a Spring Boot client application relies on the Redis Service resource being available on
 the cluster at host address `redis-server` on port `6379`. TLS support is not yet enabled, and so the definition does
-not have any configuration besides host and port of the Redis server.
+not have any configuration besides the host and port of the Redis server.
+
+The application is configured with a liveness and readiness probe. If it reconciles successfully, then you know it has
+connected to Redis successfully. 
 
-The application is configured with a liveness and readiness probe, so if it reconciles successfully, then we know it has
-connected to Redis successfully. However, for extra verification, the client application comes with an API endpoint that
-can be queried, which specifically interacts with Redis:
+For extra verification, the client application comes with an API endpoint that can be queried. 
+This specifically interacts with Redis:
 
 ```shell
 kubectl port-forward svc/spring-boot-redis-client-app 8080:8080 -n mtls-demo
 
 # In another terminal window/tab
-curl -XGET http://localhost:8080/
+curl -XGET http://localhost:8080/.
```

Comment:
**[nit]**: trailing dot may be misleading to the reader attempting to `curl` this address, if I'm not mistaken.

Reply:
Fixed.